src/components/errors/ErrorNetworkMismatch.tsx

```diff
@@ -4,12 +4,21 @@ import { getChainName } from '@dcl/schemas/dist/dapps/chain-id'
 import { ErrorContainer, ErrorDetails, ErrorImage } from './Error'
 import errorImage from '../../images/errors/robotsmiling.png'
 import { switchToChainId } from '../../eth/provider'
+import { track } from '../../utils/tracking'
 import './errors.css'
 
 export interface ErrorNetworkMismatchProps {
   extra: Record<string, any>
 }
 
+const switchChain = (options: Record<string, any>) => {
```

Comment:
Is this function going to be (possibly) needed elsewhere? it seems like it being local to this file might be a problem creating repeated code if any other piece of code needs it